libmscore/chord.cpp

```diff
@@ -2897,7 +2897,7 @@ QPointF Chord::layoutArticulation(Articulation* a)
 
 void Chord::reset()
       {
-      score()->undoChangeProperty(this, P_ID::STEM_DIRECTION, Direction(Direction::AUTO));
+      score()->undoChangeProperty(this, P_ID::STEM_DIRECTION, Direction::AUTO);
```

Comment:
so `undoChangeProperty(P_ID::STEM_DIRECTION, ds->stemDirection(pitch));` would be OK, but `undoChangeProperty(this, P_ID::STEM_DIRECTION, Direction::AUTO);` is not?